packages/backend/src/api/request.ts

```diff
@@ -128,7 +128,7 @@ export function buildRequest(options: CreateBackendApiOptions) {
       }
 
       // TODO: Parse JSON or Text response based on a response header
-      const isJSONResponse = headers['Content-Type'] === 'application/json';
+      const isJSONResponse = res?.headers && res.headers?.get('Content-Type') === 'application/json';
```

Comment:
Were we using the request headers before ?

Reply:
@panteliselef yes Panteli 😢